testsuite/features/step_definitions/command_steps.rb

```diff
@@ -978,6 +987,15 @@
   $proxy.run('firewall-offline-cmd --zone=public --add-service=mdns')
 end
 
+When(/^I copy "([^"]*)" file from "([^"]*)" to "([^"]*)"$/) do |file_path, from_host, to_host|
```

Comment:
don't we already have this  somewhere ?

Reply:
Right! I coded this before the PR from Vanda that also required that step, so it must be already somewhere! Thanks for remind me it!

Reply:
Actually, not exactly this step, what we have is a step that copy a file from the controller to some host passed by parameter.
```
When(/^I copy "([^"]*)" to "([^"]*)"$/) do |file, host|
  node = get_target(host)
  return_code = file_inject(node, file, File.basename(file))
  raise 'File injection failed' unless return_code.zero?
end
```

But in this step, we copy a file from a host passed by parameter to another host passed by parameter.

What we could do is to replace the calls to the older step, using the new step, but I think that would be the first time that we call into our Gherkin feature side a "controller" host, and that... could be a bit weird. Isn't it?

Tell me your opinion please.

Reply:
Then your solution is better. But could you please just put this new step next to the old one ?